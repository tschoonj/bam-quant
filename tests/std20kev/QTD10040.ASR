File: C:\xrfdaten\2014KW30\chris\std20kev\std10040.spe
Start: 21.07.2014 18:16:34
Stop: 21.07.2014 18:18:14
BESSY: 298.1196312 298.1150284
IONI: 3.158327E-11 3.034657E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 61923
Run: 100000
Dead: 38
ICR: 133820
OCR: 0
Accumulated BESSY: 59694.4965897
Accumulated Ioni: 6.19246269E-09
N Accumulation: 200
Normfactor:  5.2157241e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Zr
exp_x 151.260406494141 151.2604
exp_y 122.323806762695 122.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  40   1     15.746   7323791.      2707.     119.51
