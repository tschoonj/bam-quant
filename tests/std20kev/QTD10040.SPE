$DATA:
0 2000
14291
9346
4332
1477
415
98
37
22
10
9
5
4
1
0
10
20
165
1813
2726
2646
2563
2534
2391
2324
2199
2221
2165
2246
2114
2054
2133
2195
2012
2181
2134
2122
2068
2021
2051
2105
2092
2050
1939
1867
1801
1853
1734
1693
1622
1556
1496
1409
1437
1385
1324
1258
1220
1188
1120
1188
1160
1087
1111
1100
1049
978
1008
953
965
869
931
913
896
914
867
871
839
870
774
738
770
812
773
751
787
780
821
817
738
800
755
786
775
780
759
814
990
1327
1755
2360
3008
3388
3578
3490
3367
3051
2849
2295
1904
1591
1316
1084
838
738
707
633
625
621
593
583
582
589
547
574
625
599
621
636
631
627
589
607
586
583
566
566
544
526
560
556
543
562
527
549
609
545
587
593
607
536
518
519
494
492
505
529
506
495
491
503
480
476
489
489
486
483
468
486
443
489
471
480
494
457
482
469
464
480
451
459
476
486
476
481
498
453
471
484
488
453
458
435
463
483
453
415
466
470
469
416
469
452
436
454
457
481
400
444
426
440
461
425
473
424
435
433
434
444
440
408
428
409
429
447
426
435
427
441
436
427
414
419
420
443
417
448
464
437
405
437
447
432
412
461
421
431
405
453
439
407
407
420
404
389
403
418
365
400
388
420
412
430
449
376
408
428
463
422
448
436
412
460
455
426
402
384
400
393
407
421
397
424
360
351
418
425
410
379
376
397
392
427
418
430
393
404
418
421
374
404
378
380
378
374
407
364
410
416
414
378
401
478
464
539
548
632
671
645
649
589
533
514
482
396
374
392
379
405
406
375
376
372
381
361
403
416
416
413
425
368
404
385
452
427
421
467
414
450
466
460
443
465
428
397
422
383
374
424
406
394
401
393
401
364
413
403
394
401
409
420
428
440
403
428
415
401
429
372
376
414
414
390
459
475
523
714
765
978
1115
1410
1510
1504
1444
1334
1116
866
742
622
526
484
441
418
428
435
435
430
394
409
428
387
383
404
383
419
401
365
384
364
423
372
396
388
393
359
431
408
384
377
376
395
396
390
368
360
429
408
478
505
586
651
692
791
891
1069
1282
1352
1390
1385
1363
1259
1164
1042
875
787
775
680
663
639
616
622
642
682
642
656
685
646
521
516
506
483
435
445
422
437
414
404
400
438
440
386
404
399
395
410
424
398
408
403
393
400
419
437
381
418
386
372
414
414
403
388
406
406
411
406
423
422
434
450
421
444
397
446
452
463
469
485
489
535
610
575
673
617
589
596
595
534
534
517
458
461
403
480
526
527
552
502
537
542
501
566
489
482
519
502
469
442
422
441
426
460
450
442
450
421
444
498
473
446
480
447
463
463
423
463
508
489
428
482
471
457
457
463
449
458
482
495
464
489
494
494
512
509
478
483
477
480
462
519
491
501
499
523
477
504
532
502
511
529
497
514
504
497
511
486
496
551
524
543
557
545
580
542
548
519
549
544
503
600
560
516
583
575
562
594
567
616
546
598
630
593
564
598
586
598
584
589
596
606
634
603
667
609
657
640
657
655
655
642
642
645
693
636
675
680
639
669
615
714
618
685
699
663
648
676
660
649
676
639
677
659
682
701
647
698
723
724
704
722
703
759
772
796
784
808
860
889
892
967
1045
1096
1164
1262
1294
1273
1337
1274
1251
1249
1169
1054
1085
1000
979
1002
1027
988
979
1010
1031
1029
986
1097
1148
1126
1155
1150
1209
1233
1273
1333
1366
1332
1391
1491
1482
1543
1525
1642
1705
1757
1827
1866
1892
1972
2027
2123
2261
2249
2335
2377
2365
2411
2528
2658
2674
2696
2794
2874
2853
2974
3070
3215
3201
3283
3489
3409
3661
3758
3937
4152
4397
5021
5542
6933
9086
12854
18890
28720
43486
65659
95788
135796
186890
246315
314625
385930
453779
513929
559816
588711
590230
568296
525948
465830
393833
319230
247523
183348
130087
88258
57964
36723
22689
14169
8780
5743
4064
3093
2373
2144
1897
1698
1581
1405
1349
1330
1235
1229
1195
1099
1099
1076
981
979
1024
968
927
928
957
984
921
966
988
906
936
910
909
940
900
920
920
897
885
939
949
922
954
1002
989
1003
1048
985
1074
1083
1093
1139
1103
1168
1239
1273
1305
1288
1464
1365
1449
1509
1589
1717
1884
2153
2658
3345
4469
6342
9096
12967
18724
26163
34626
44942
55585
66404
75027
80814
84933
83856
79646
73015
62789
52617
42682
33906
26245
20786
17254
14703
13572
13477
13214
12936
12276
11565
10501
9194
7611
6040
4488
3405
2304
1653
1043
707
455
311
233
164
115
120
82
86
87
70
71
73
59
67
58
60
43
62
58
53
42
39
45
48
30
39
55
45
37
30
34
35
35
32
37
29
32
33
42
26
53
29
36
40
37
34
19
44
29
33
36
34
28
39
29
30
34
32
48
25
28
34
42
38
46
40
41
45
42
60
42
45
49
72
63
53
52
48
74
68
73
72
83
76
83
65
103
99
86
69
73
83
56
56
59
63
52
44
33
19
40
31
32
20
22
33
18
15
20
15
14
16
15
16
10
21
8
18
6
18
8
16
9
13
13
8
18
11
20
11
10
9
9
18
14
12
16
16
12
12
9
15
13
10
13
14
14
15
9
16
10
11
7
9
13
5
11
3
10
11
6
14
9
6
11
8
11
8
15
9
9
8
9
7
13
4
12
13
9
11
12
8
12
8
16
12
13
9
19
11
9
21
14
13
10
11
9
16
11
10
15
7
16
13
8
12
6
10
12
6
13
9
9
8
6
12
13
8
12
7
14
11
11
15
9
5
12
14
19
16
10
10
6
11
8
9
6
14
10
9
9
14
8
11
13
11
12
7
6
13
10
13
9
11
10
13
9
14
17
16
20
16
25
21
26
19
21
28
21
14
27
21
19
23
19
19
21
20
10
12
8
8
9
6
6
16
16
11
9
13
6
12
11
11
12
10
13
9
20
8
6
9
12
11
8
10
17
14
21
11
13
15
13
16
11
24
15
19
23
27
17
23
15
34
26
26
22
24
15
19
17
26
18
23
21
23
16
20
12
17
10
11
10
7
16
11
17
10
7
7
9
10
10
15
12
12
11
12
15
12
11
13
17
12
7
14
9
12
11
13
14
6
11
6
7
10
13
14
7
10
11
6
16
12
7
6
9
10
8
11
15
11
11
17
13
9
11
8
13
10
17
21
12
14
12
7
9
14
14
11
11
8
20
11
10
10
8
17
13
11
8
11
18
11
8
12
8
14
13
7
11
15
9
7
12
15
13
13
12
12
8
4
13
15
14
11
8
11
13
14
4
4
9
14
12
13
13
7
10
3
10
11
14
9
10
7
8
9
14
20
10
12
8
9
13
9
13
12
13
9
12
8
10
12
16
11
12
12
11
13
9
9
15
9
19
8
7
15
7
12
13
10
18
14
15
8
12
9
6
15
18
11
16
21
18
11
11
18
13
20
18
13
22
16
14
13
13
20
19
9
20
17
16
14
9
20
13
16
14
14
16
11
12
11
10
17
10
13
14
21
21
18
15
19
14
17
12
15
13
18
21
17
21
9
29
19
27
20
21
20
24
28
24
11
16
25
26
21
18
15
15
26
26
12
26
32
21
24
29
25
26
27
21
26
44
28
27
30
29
45
15
24
37
34
43
34
39
34
40
37
35
29
41
61
55
52
47
58
56
47
64
49
64
72
60
58
59
68
78
84
74
101
107
144
157
208
212
359
443
536
667
915
1100
1288
1557
1954
2279
2623
3014
3314
3749
3842
4309
4433
4585
4621
4572
4388
4196
3993
3720
3188
2711
2362
2010
1685
1327
1044
815
627
496
360
272
210
123
100
79
63
61
52
35
36
30
36
23
14
24
27
25
18
23
19
20
13
24
22
24
24
26
24
30
19
32
26
20
27
23
26
27
16
23
23
26
27
31
32
30
33
24
26
30
37
30
41
38
45
44
48
69
57
85
119
127
160
182
224
289
336
409
482
592
650
785
827
981
1069
1140
1230
1199
1283
1278
1275
1307
1218
1099
1095
923
833
726
645
578
456
443
425
324
282
273
218
192
164
160
142
115
86
79
83
61
48
32
19
17
8
14
9
7
8
8
8
7
3
3
4
5
3
5
6
4
5
3
6
3
1
2
2
7
3
7
5
3
2
3
5
3
1
4
5
8
3
4
6
8
6
1
4
7
9
8
9
14
15
27
30
30
38
57
32
56
74
59
48
80
80
97
94
98
90
92
88
85
83
81
71
74
55
49
48
43
49
45
46
26
29
36
28
24
29
16
9
19
10
20
3
9
7
7
4
1
2
5
3
2
3
2
4
3
1
3
0
0
1
0
2
0
0
0
1
0
0
0
0
1
0
1
0
1
0
0
1
0
0
1
1
0
0
0
0
0
1
0
0
0
0
1
1
2
0
0
1
0
0
0
0
0
1
0
0
0
0
0
0
0
1
0
0
1
0
0
0
0
0
0
0
1
0
0
0
1
1
1
0
1
1
0
0
1
0
2
2
0
0
0
2
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
2
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
0
0
1
2
0
0
1
0
0
0
0
0
1
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
1
1
0
1
0
0
0
0
0
1
0
0
0
0
0
0
0
0
1
1
0
0
0
0
0
1
2
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10040.spe
Start: 21.07.2014 18:16:34
Stop: 21.07.2014 18:18:14
BESSY: 298.1196312 298.1150284
IONI: 3.158327E-11 3.034657E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 61923
Run: 100000
Dead: 38
ICR: 133820
OCR: 0
Accumulated BESSY: 59694.4965897
Accumulated Ioni: 6.19246269E-09
N Accumulation: 200
Normfactor:  5.2157241e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Zr
 
exp_x 151.260406494141 151.2604
exp_y 122.323806762695 122.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  40   1     15.746   7323791.      2707.     119.51
