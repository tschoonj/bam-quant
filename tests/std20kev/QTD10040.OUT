     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:23:45  บ   
     บ  Spectrum: QTD10040.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        4 iterations done    บ   
     บ  ChiSquare =   176.2   last change =   .02%    lambda= 1.E-06   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -61.9 ๑   100.0   บ     -61.9 ๑      .4   บ   
     บ  GAIN (eV/ch)  บ     20.255 ๑   2.000   บ    20.255 ๑    .001   บ   
     บ det NOISE (eV) บ     -137.5 ๑    40.0   บ    -137.5 ๑    -2.4   บ   
     บ  FANO factor   บ       .198 ๑    .050   บ      .198 ๑    .001   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Zr-Ka                           7323791. ๑   2707.           บ   
     บ    KA1        15.775    .65677     4810061. ๑   1778.    134.4  บ   
     บ              781.879    218.93        1542.           9.88E-01  บ   
     บ    KA2        15.691    .34323     2513730. ๑    930.    112.2  บ   
     บ              777.732    218.11        1566.           9.88E-01  บ   
     บ                                                                 บ   
     บ  2 Zr-Kb                           1155436. ๑   1076.           บ   
     บ    KB1        17.663    .87511     1011128. ๑    942.    162.5  บ   
     บ              875.091    236.49        1035.           9.81E-01  บ   
     บ    KB2        17.970    .12489      144308. ๑    138.     18.7  บ   
     บ              890.247    239.23        1063.           9.79E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                          112410. ๑    337.           บ   
     บ    L1O3       31.492    .74597       83880. ๑    252.      8.2  บ   
     บ             1557.853    338.42          98.           5.82E-01  บ   
     บ    L1O2       33.409    .20604       23168. ๑     70.      2.6  บ   
     บ             1652.489    350.22         133.           5.27E-01  บ   
     บ    L1M5       33.716    .02941        3307. ๑     15.      2.2  บ   
     บ             1667.645    352.07         137.           5.19E-01  บ   
     บ    L1M4       35.326    .01423        1600. ๑     13.      1.4  บ   
     บ             1747.124    361.63         155.           4.78E-01  บ   
     บ    L1N5       35.633    .00406         457. ๑     12.      2.2  บ   
     บ             1762.280    363.42         153.           4.71E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
